utils/index.d.ts

```diff
@@ -2100,3 +2123,239 @@ export declare namespace validationUtils {
      */
     export function getInvalidCharLengthMessage(constraints?: RangeConstraints): string;
 }
+
+// #region agent input types
+
+export type ParameterAgentMetadata = {
+    /**
+     * A camel cased string that names the parameter this quick pick is for. Will be given to a LLM to identify the
+     * parameter that it needs to pick a value for.
+     *
+     * For example:
+     * - "subscription"
+     * - "resourceGroup"
+     * - "runtime"
+     * - "name"
+     */
+    parameterName: string;
+
+    /**
+     * A title cased string for the parameter this quick pick is for. Will be displayed to the user.
+     *
+     * For example:
+     * - "Subscription"
+     * - "Resource Group"
+     * - "Runtime"
+     * - "Name"
+     */
+    parameterDisplayTitle: string;
```

Comment:
Could we consolidate `parameterName` and `parameterDisplayTitle`? It's not a real big deal, but I can't really think of a scenario where they would need to be different and it just adds more boilerplate for each step.

Or maybe we can make one option, and it would just set itself equal to whatever the required one is if the property wasn't provided?

Reply:
I will try dropping `parameterName` for now. Looking at the agent code I *think* it should be ok. It may need to come back though.